server/parsec/components/postgresql/events.py

```diff
@@ -29,44 +27,32 @@
 
 
 class PGEventBus(EventBus):
-    def __init__(self, send_events_channel: MemoryObjectSendStream[Event]):
+    def __init__(self, conn: AsyncpgConnection):
         super().__init__()
-        self._send_events_channel = send_events_channel
+        self._conn = conn
 
     @override
     async def send(self, event: Event) -> None:
```

Comment:
since the method is only for testing, better to name it with a `test_` prefix so that it pops out during code review whenever it is wrongly used.

```suggestion
    async def test_send(self, event: Event) -> None:
```

(I guess you couldn't do it because it was used in the `server/tests` code where `EventBus` can be either memory or postgresql, if so it's probably not worth it to bother with this change ^^)

Reply:
55c12c1d0e384293ef21a8d0a192af44d5650931